### PR TITLE
FORMS-225 # make manual calculations manual again

### PR DESCRIPTION
**Pull request by GeetaSajwan**
- manual calculation button now listen to change: value
- updated test

### CODE REVIEW

File: forms/models/behaviour.js [hunk 1]
```diff
@@ -125,7 +125,7 @@ define(function (require) {
       }
 
       // legacy calculations only expect to monitor value changes (performance)
```
Comment (jokeyrhyme) [inline forms/models/behaviour.js]: It's probably worth getting rid of this comment, too. As it no longer describes what happens now.

Comment (GeetaSajwan) [inline forms/models/behaviour.js]: yes, you are right so I removed that comment.

**Comment by simonmarklar** [comment]
lgtm :+1:

**Comment by jokeyrhyme** [comment]
I changed the title of this PR to match the commit format expected by JIRA. Otherwise, JIRA suspects that we didn't merge the code. JIRA is being silly. /shrug

**Pull request merged**